week2/olya/sentence_validator.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+def sentence_validator(text):
+    """
+    Check  if all sentences in the text are valid
+        Args: string
+        Returns: true or false
+    """
+
+    pattern = "[.?!]"
+    sentences = re.split(pattern, text)
+
+    for sentence in sentences[:-1]:
+        word = sentence.strip()
+        if word[0].islower():
+            return False
+    for sentence in sentences[:-1]:
+        comma_pattern = ",(?=\S)|:"
```

Comment:
Do you need this definition inside the for loop???

Reply:
No, i will place it outside of the loop